app/src/main/res/values/urls.xml

```diff
@@ -0,0 +1,3 @@
+<resources>
+    <string name="relaynet_website">https://relaynet.network/</string>
```

Comment:
Can you please make this so localised websites can be used? I'd like to point users on the `es-*` locale to https://relaynet.red/ instead, for example.

Reply:
That's already localizable. If you create a file `values-es/urls.xml` you can write the spanish version there.

Reply:
Nice!